### PR TITLE
Implement Zoom UI in Hamburger Menu

### PR DESCRIPTION
**Pull request by mshin-wolvic**
Unlike the general menu items with ImageView&TextView layout, a Zoom menu has ImageView&TextView&+/- ImageViews layout and only +/- ImageViews should handle hover and click events.
This patch configures the zoom layout and add the relevant code. The +/- ImageViews are connected to the page zoom feature, and this Zoom UI appears only in chromium-backend.

### CODE REVIEW

File: app/build.gradle [hunk 1]
```diff
@@ -130,6 +130,8 @@ android {
         buildConfigField "String[]", "SPEECH_SERVICES", "{ com.igalia.wolvic.speech.SpeechServices.MEETKAI }"
         buildConfigField "Boolean", "SUPPORTS_SYSTEM_NOTIFICATIONS", "false"
         buildConfigField "Float", "DEFAULT_DENSITY", "1.25f"
+        buildConfigField "Boolean", "ENABLE_PAGE_ZOON", "false"
```
Comment (zakharvoit) [inline app/build.gradle:133]: ZOOM

**Review by zakharvoit — CHANGES_REQUESTED** [review]
Left one nit, but I'd like somebody else from the team to review since I'm not very familiar with wolvic/ code.

**Review by svillar — COMMENTED** [review]
We are not adding translations in PRs anymore because that messes up Weblate, so let's just include the new strings in strings.xml and remove the translations.

**Comment by svillar** [comment]
Ah and another comment, please add a '%' after the zoom level in the UI, otherwise seeing 100, 175 etc looks weird

**Comment by mshin-wolvic** [comment]
> We are not adding translations in PRs anymore because that messes up Weblate, so let's just include the new strings in strings.xml and remove the translations.

Done.

> Ah and another comment, please add a `%` after the zoom level in the UI, otherwise seeing 100, 175 etc looks weird

Done.

Thanks. :)

**Review by svillar — APPROVED** [review]
Wonderful!

**Pull request merged**